vendor/riscv/riscv-isa-sim/riscv/Proc.cc

```diff
@@ -37,33 +59,136 @@ st_rvfi Processor::step(size_t n, st_rvfi reference) {
   rvfi.rs2_addr = this->get_state()->last_inst_fetched.rs2();
   // TODO add rs2_value
 
-  rvfi.trap = this->taken_trap;
-  rvfi.trap |= (this->which_trap << 1);
 
+  if(this->next_rvfi_intr){
```

Comment:
Also curious about this script, and if other parts of the code modify cva6 execution

Reply:
Since there has been more than a week since I notified that "we only have one week" and there has been 0 external involvement since then, I think we can justify assuming that nothing breaks, particularly since this is on a 40s-specific branch so the rest of the system is shielded for now.
Additional testing will need to be done for a future merge to master.